jcl/src/openj9.dtfj/share/classes/module-info.java

```diff
@@ -37,6 +37,7 @@
   /*[IF PLATFORM-mz31 | PLATFORM-mz64]*/
   requires ibm.jzos;
   /*[ENDIF]*/
+  exports com.ibm.j9ddr.tools.ddrinteractive to openj9.dtfjview;
```

Comment:
Seems ddrinteractive belongs in dtfjview instead of dtfj.

Reply:
Perhaps, but that's a much larger refactoring that I suggest should be done separately - it would involve partitioning `debugtools/DDR_VM/src/com/ibm/j9ddr` into parts for dtfj and parts for dtfjview: it's not immediately obvious to me that is necessarily possible.